src/global.ts

```diff
@@ -259,6 +259,16 @@ export interface Auth0ClientOptions extends BaseLoginOptions {
    * **Note**: Using this improperly can potentially compromise the token validation.
    */
   nowProvider?: () => Promise<number> | number;
+
+  /**
+   * If provided, the SDK will load the token worker from this URL instead of the integrated `blob`. An example of when this is useful is if you have strict
+   * Content-Security-Policy (CSP) and wish to avoid needing to set `worker-src: blob:`. We recommend either serving the worker, which you can find in the module 
+   * at `<module_path>/dist/auth0-spa-js.worker.production.js`, from the same host as your application or using the Auth0 CDN 
+   * `https://cdn.auth0.com/js/auth0-spa-js/<version>/auth0-spa-js.worker.production.js`.
+   * 
+   * **Note**: The worker is only used when `useRefreshTokens: true` and no custom `cache` is provided.
```

Comment:
I think this is not correct. Based on the [code](https://github.com/auth0/auth0-spa-js/blob/main/src/Auth0Client.ts#L232), it should be only when useRefreshTokens is true, and the cache is in memory (so not local-storage or custom).

Reply:
Thanks for catching that! Just made an update to it in [03fb78d](https://github.com/auth0/auth0-spa-js/pull/1208/commits/03fb78d6aaefdf2c4b4f7322c5acf237a477dbe7).